Flutter/calendar/appointments.md

```diff
@@ -130,6 +130,86 @@ events.notifyListeners(CalendarDataSourceAction.reset, null);
 {% endhighlight %}
 {% endtabs %}
 
+## Notify listener
+
+By call this method whenever the datasource changes, to notify any clients the datasource may have. We have provided support to perform three operations like Add, Remove and Reset. NotifyListener is used to notify the dataSource may have changed.
```

Comment:
We have provided support to perform three operations like Add, Remove and Reset. NotifyListener is used to notify the dataSource may have changed.

Need to notify the datasource action(add, remove and reset) through notify listener

Reply:
As discussed, I have checked and updated the changes in my latest commit.